client/lib/url/index.js

```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import startsWith from 'lodash/string/startsWith';
+import support from './support';
```

Comment:
I'm sure you know this, but just pointing out that this should be in an `Internal dependencies` block.